charts/newrelic-logging/templates/_helpers.tpl

```diff
@@ -153,12 +153,24 @@ Returns fargate
 Returns lowDataMode
 */}}
 {{- define "newrelic-logging.lowDataMode" -}}
-{{- if .Values.global }}
-  {{- if .Values.global.lowDataMode }}
-    {{- .Values.global.lowDataMode -}}
+{{/* `get` will return "" (empty string) if value is not found, and the value otherwise, so we can type-assert with kindIs */}}
+{{- if (get .Values "lowDataMode" | kindIs "bool") -}}
+  {{- if .Values.lowDataMode -}}
+    {{/*
+        We want only to return when this is true, returning `false` here will tamplate "false" (string) when doing
+        an `(include "newrelic-logging.lowDataMode" .)`, whis is not an "empty string" so it is `true` if it is used
+        as an evaluation somewhere else.
+    */}}
+    {{- .Values.lowDataMode -}}
+  {{- end -}}
+{{- else -}}
+{{/* This allows us to use `$global` as an empty dict directly in case `Values.global` does not exists */}}
```

Comment:
I am scared of this comment inside the template, as it will render a newline.

Reply:
No. It will not. And it was tested before committing ;)